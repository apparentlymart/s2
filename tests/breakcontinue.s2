#-*-c++-*-

layerinfo "type" = "core";
layerinfo "majorversion" = 1;

function main () {
	var string[] test = [ "Hello", "I", "am", "a", "goat" ];

    foreach var string item ($test) {
        if ($item == "am") {
            break;
        }
        println($item);
    }

    foreach var string item ($test) {
        if ($item == "am") {
            continue;
        }
        println($item);
    }


}

