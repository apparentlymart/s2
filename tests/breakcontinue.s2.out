Hello
I
Hello
I
a
goat
